model/src/main/proto/arguments.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// The fragment arguments passed to instances of ExitProfileDialogFragment.
+message ExitProfileDialogArguments {
+  // Decides the correct menu item to be highlighted after canceling the ExitProfile dialog.
+  HighlightItem highlight_item = 1;
+}
+
+// Represents the type of item/menuItem that should be highlighted after canceling the
+// ExitProfile dialog.
```

Comment:
This format is incorrect for `ExitProfile dialog`.
Suggest writing this as `ExitProfileDialogFragment`

Reply:
Done.